stripper/ze_l0v0l_v1_4_csgo2.cfg

```diff
@@ -28,4 +28,58 @@ modify:
 	{
 		"item_assaultsuit" "1"
 	}
-}
\ No newline at end of file
+}
+
+;fix level 4 nuke from killing every single player after the Nuke King boss is dead and you get teleported
+modify:
+{
+	match:
+	{
+		"targetname" "lvl4_win"
+	}
+	delete:
+	{
+		"OnStartTouch" "Map_BrushRunScriptCodeKillAll();61"
+	}
+	insert:
+	{
+		"OnStartTouch" "!activatorAddOutputhealth 99999999990-1"
+	}
+}
+
+	add:
+	{
+		"model" "*505"
+		"targetname" "lvl4_winfix"
+		"StartDisabled" "1"
+		"spawnflags" "1"
+		"origin" "-4487 5370 7350.65"
+		"filtername" "filter_zombie"
+		"classname" "trigger_multiple"
+		"OnStartTouch" "!activatorAddOutputhealth 99999999990-1"
+	}	
+modify:
+	{
+	match:
+	{
+		"targetname" "endrelay4"
+		"classname" "logic_relay"
+	}
+	insert:
+	{
+		"OnTrigger" "lvl4_winfixEnable01"
+	}
+}
```

Comment:
I don't see how this will fix the nuke bug you described? You've taken out the KillAll() function which was the "nuke", but you've only added 2 new spots where people will get 9999999999 health and nothing else.

Under this current logic, my understanding is that nobody will die while people at the final area get set to 9999999999 health.

Reply:
I have always seen it unintended by the mapper that you would die at the end of stage 4 after you get teleported, after the message>>> FINALLY YOU ESCAPED FROM QUEEN <<< you would die from this, it effects every single player in the teleporter no matter if you are a CT or T, it would kill you at the teleporter after nuke goes off, the nuke output exists at the position, which is why humans can get killed from the nuke at the final teleporter.

Reply:
That's not what I'm talking about, I mean that there's no nuke anymore after this change, even for zombies. The round would never end.

If you wanted to fix that bug, you'll have to either correct the logic for KillAll() to check team, or make a new nuke.